doc/source/reference/random/c-api.rst

```diff
@@ -112,7 +104,7 @@ The functions are named with the following cconventions:
 
 .. c:function:: double random_f(bitgen_t *bitgen_state, double dfnum, double dfden)
 
-.. c:function:: double random_standard_cauchy(bitgen_t *bitgen_state)
+.. c:function:: double random_cauchy(bitgen_t *bitgen_state)
```

Comment:
If it already matches the name at the Python level, please leave it as such.

Reply:
Reverting the cauchy and random_t changes.

Reply:
@mattip I think you forgot to push the revert commit here.

Reply:
good catch. fixing.